pkg/gen/python.go

```diff
@@ -94,6 +96,21 @@ func PythonClient(
 		if err != nil {
 			return err
 		}
+		if group.Group() == "apiextensions" {
+			groupInitPy += fmt.Sprint(`
+from .CustomResource import *
+`)
```

Comment:
nit: fix formatting

Reply:
I wanted newlines before and after in the raw string, so that's correct. (Although admittedly weird looking)